tajo-docs/src/main/sphinx/table_management/compression.rst

```diff
@@ -1,5 +1,23 @@
-*********************************
+***********
 Compression
-*********************************
+***********
 
-.. todo::
\ No newline at end of file
+Using compression makes data size compact and network traffic low. Most of Tajo data types support data compression feature.
```

Comment:
You may mean ```data formats``` instead of ```data types```. 
Also, ```network traffic low``` seems to be ambiguous. How about changing the first sentence to ```Using compression can make data size compact, thereby enabling efficient use of network bandwidth and storage.```?